docs/docs/gatsby-cli.md

```diff
@@ -23,38 +22,38 @@ You can also use the `package.json` script variant of these commands, typically
 }
 ```
 
-## API commands
+## एपीआई कमांड
 
-### `new`
+### `नया`
 
 ```
 gatsby new [<site-name> [<starter-url>]]
 ```
 
-#### Arguments
+#### तर्क
 
-| Argument    | Description                                                                                                                                                                                                     |
+| तर्क    | विवरण                                                                                                                                                                                                     |
 | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| site-name   | Your Gatsby site name, which is also used to create a project directory.                                                                                                                                        |
-| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/gatsby-starters/) docs for more information. |
+| site-name   | आपका gatsby साइट का नाम, जिसका उपयोग प्रोजेक्ट डायरेक्टरी बनाने के लिए भी किया जाता है।                                                                                                                                        |
+| starter-url | एक gatsby स्टार्टर URL या स्थानीय फ़ाइल पथ। के लिए चूक [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); देखें [Gatsby starters](/docs/gatsby-starters/) अधिक जानकारी के लिए डॉक्स। |
```

Comment:
`देखें [Gatsby starters](/docs/gatsby-starters/) अधिक जानकारी के लिए डॉक्स।` => `और अधिक जानकारी के लिए [Gatsby starters](/docs/gatsby-starters/) डॉक्स को देखें।`

Reply:
`डॉक्स को देखें।` => `डॉक्स को देखें। |`

you missed ending line for this